scene/gui/color_picker.cpp

```diff
@@ -564,7 +564,9 @@ void ColorPicker::_html_submitted(const String &p_html) {
 	}
 
 	const Color previous_color = color;
-	color = Color::from_string(p_html, previous_color);
+	// Remove whitespace from the HTML color code which can easily occur
+	// when copy-pasting text from color palettes on the internet.
```

Comment:
I think the comment is uneeded, pretty self explanatory

Reply:
I tried to explain why I changed that line, but after reading through my comment again... Yep, I must admit that this is quite a repetition.